lib/config/index.js

```diff
@@ -211,7 +211,8 @@ function checkClusterOptions (config) {
 
   assert(typeof cfg.ipv6 === 'boolean', '[FATAL] kuzzlerc.cluster.ipv6: boolean expected');
 
-  assert(typeof cfg.ip === 'string', '[FATAL] kuzzlerc.cluster.ip: value must be a string');
+  assert(!cfg.ip || ['private', 'public'].includes(cfg.ip), '[FATAL] kuzzlerc.cluster.ip: invalid value (accepted values: public, private)');
```

Comment:
Same comment here about the error message prefix but maybe we could have a `[CONFIG]` prefix?

I know it's not quite related to the scope of this PR so we can first talk about it and then do it in a different PR.
I started to think about this when I was trying to debug one of our client with a huge log file and I was trying to find only cluster related messages.